yogstation/code/modules/mob/living/simple_animal/friendly/clownkrub.dm

```diff
@@ -1,5 +1,5 @@
 /mob/living/simple_animal/crab/clown
-	name = "Mr Krebs"
+	name = "Bananium Kreb"
```

Comment:
No thanks

Reply:
Im a bit confused on this one but the reason I did it was because I wanted to have Mr Krebs be in the clown facility but couldnt think of any other good names so to make it more unique I made the regular kreb more generic in comparison